eva-accession-release-automation/gather_clustering_counts/gather_clustering_counts_from_mongo.py

```diff
@@ -292,9 +390,9 @@ def fill_data_for_current_release(metadata_connection_handle, metrics_per_assemb
 
         # Finally perform inserts, once for each taxonomy but otherwise identical
         for taxid, scientific_name in ranges_per_assembly[asm]['species_info']:
-            folder = f"{scientific_name}/{asm}"
-            formatted_name = scientific_name.capitalize().replace('_', ' ')
-            insert_query = f"insert into {assembly_table_name} "\
+            folder = f"{scientific_name}/{asm}".replace("'", "''")
```

Comment:
Just curious, why are there sometimes quotes in the scientific name?

Reply:
Not sure. This is something I observed with taxonomy 8307 where the scientific name is "Ambystoma 'unisexual hybrid'"